### DISCUSSION TITLE
Table Actions not working

### DISCUSSION THREAD

**Discussion by dharmarn** [filamentphp/filament#15091] [Help] [answered]
### Package

Actions

### Package Version

v3.2

### How can we help you?

In my filament project recently, I have installed the liveware through bellow comment
-> composer require livewire/livewire .
After this only I can see the liveware version in my composer.son file below is the details.

   "php": "^8.1",
        "barryvdh/laravel-dompdf": "^2.1",
        "filament/filament": "3.2",
        "guzzlehttp/guzzle": "^7.2",
        "laravel/framework": "^10.10",
        "laravel/sanctum": "^3.3",
        "laravel/tinker": "^2.8",
        "livewire/livewire": "^3.5",
        "maatwebsite/excel": "^3.1",
        "pxlrbt/filament-activity-log": "^1.0",
        "pxlrbt/filament-excel": "^2.3",
        "spatie/laravel-permission": "^6.3"

After this View, Delete Tables actions and Table Tab click actions are not working. It's showing black screen .
Please help me to fix this issue.

**Comment by leandrocfe** [comment, marked as answer]
change `"filament/filament": "3.2"` to `"filament/filament": "^3.2"`

run `composer update -W`

**Reply by dharmarn** [reply]
Thank you so much ..
It's working..

**Reply by Ahirusha11** [reply]
Thanks :)

**Reply by YordanisPonce** [reply]
This is the answer. Thank you so much!!!!!

**Reply by LucaSteinauer** [reply]
I tried your solution but for me its still not fixing the problem.
The modals are working only with forms but not with tables.
Any idea of what it could be? @leandrocfe

**Reply by sylainx** [reply]
don't work for me

**Reply by NaufalParamaRafif** [reply]
Thanks!!!

**Reply by zhairiazzeddine** [reply]
Thanks Broo

**Comment by Hashim-svg** [comment]
after this modals/pop ups not working 
![image](https://github.com/user-attachments/assets/a0c66ba0-e540-4d98-85c7-74ff9626529d)

**Reply by leandrocfe** [reply]
`php artisan filament:assets`
?

**Reply by Hashim-svg** [reply]
not worked
even deleted bulk action is not appears correct 
![image](https://github.com/user-attachments/assets/46c0da51-aa77-4fae-a381-372a47da6d2b)

**Reply by Hashim-svg** [reply]
![image](https://github.com/user-attachments/assets/6879ba14-529c-49a2-98c4-509894873ce4)

**Reply by Hashim-svg** [reply]
i have created new project moved models , migration, controllers , filament resources ...... but the problem was still there , than i have applied 
change "filament/filament": "3.2" to "filament/filament": "^3.2"

run composer update -W
php artisan filament:assets

for new filamentphp project i used command ": composer require filament/filament:"^3.2" -W
@leandrocfe  thanks for support <3 Great guy

**Reply by leandrocfe** [reply]
`php artisan about --only=Filament`

What is the output?

**Reply by Hashim-svg** [reply]
on new and old project same result but new one is working and old has still same issues
![image](https://github.com/user-attachments/assets/2ca4362c-3fc9-478f-9176-eacd85d5f038)

**Reply by leandrocfe** [reply]
Did you publish views?

**Reply by Hashim-svg** [reply]
Yes!

**Reply by tonypartridge** [reply]
> Yes!

Please unpublish all filament views, these are advised not to publish unless absolutely necessary.  IF you do need too, check all the changes between your views and the filament views and merge any changes.

**Reply by tonypartridge** [reply]
then also run npm run build

**Comment by tonypartridge** [comment]
Have you overwritten any views?

Many thanks
Tony

**Reply by LucaSteinauer** [reply]
https://github.com/user-attachments/assets/916194c2-16c8-47ee-a0d1-977ee0e7a593


This is the behavior of the problem.
i didnt overwrite anything

**Reply by clairenakilacc** [reply]
Same problem here. I already tried the answers above but remains unresolved.

**Reply by LucaSteinauer** [reply]
> Same problem here. I already tried the answers above but remains unresolved.

Can you try to type in the blade component where u want to show your table:

<x-filament-actions::modals />
{{ $this->table }}

it should work.

**Reply by clairenakilacc** [reply]
I use the Filament resources I do not have blade. even when I search anything, a black screen appears, like what is being shown in the video

**Reply by bdj-Razik** [reply]
@LucaSteinauer  hey bro please any solution for this issue

**Reply by furkanbardakk** [reply]
hey, any solution for this issue

**Comment by studiowizjo** [comment]
I am experiencing the same issue. I have tried the following troubleshooting steps:

- Disabling everything possible in `AdminPanelProvider`
- Clearing cache
- Removing various packages, updating Filament, or downgrading to `3.1.0`
- Running `npm run build`

I have not published any Filament views.

I am out of ideas on what could be causing this issue.

---

### Output of `php83 artisan about` 

```
Environment
  Application Name ........................................................................................................................... [CUT]
  Laravel Version ......................................................................................................................... 10.48.27
  PHP Version ............................................................................................................................... 8.3.15
  Composer Version ............................................................................................................................... -
  Environment ................................................................................................................................ local
  Debug Mode ............................................................................................................................... ENABLED
  URL ............................................................................................................................ [CUT]
  Maintenance Mode ............................................................................................................................. OFF

Cache
  Config ................................................................................................................................ NOT CACHED
  Events ................................................................................................................................ NOT CACHED
  Routes ................................................................................................................................ NOT CACHED
  Views ..................................................................................................................................... CACHED

Drivers
  Broadcasting ................................................................................................................................ null
  Cache ....................................................................................................................................... file
  Database ................................................................................................................................... mysql
  Logs .............................................................................................................................. stack / single
  Mail ........................................................................................................................................ smtp
  Queue ....................................................................................................................................... sync
  Session ..................................................................................................................................... file

Filament
  Blade Icons ........................................................................................................................... NOT CACHED
  Packages ......................................................................................... filament, forms, notifications, support, tables
  Panel Components ...................................................................................................................... NOT CACHED
  Version ................................................................................................................................. v3.2.136
  Views .............................................................................................................................. NOT PUBLISHED

Livewire
  Livewire ................................................................................................................................. v3.5.12
```

---

### `composer.json`

```json
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The skeleton application for the Laravel framework.",
    "keywords": ["laravel", "framework"],
    "license": "MIT",
    "require": {
        "php": "^8.1",
        "awcodes/filament-quick-create": "^3.2",
        "filament/filament": "^3.1.0",
        "filament/spatie-laravel-media-library-plugin": "3.2",
        "guzzlehttp/guzzle": "^7.2",
        "laravel/framework": "^10.10",
        "laravel/tinker": "^2.8",
        "mohamedsabil83/filament-forms-tinyeditor": "^2.0",
        "opcodesio/log-viewer": "^2.2",
        "stechstudio/filament-impersonate": "^3.5",
        "stripe/stripe-php": "^16.5"
    },
    "require-dev": {
        "fakerphp/faker": "^1.9.1",
        "laravel/pint": "^1.0",
        "laravel/sail": "^1.18",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^7.0",
        "phpunit/phpunit": "^10.1",
        "spatie/laravel-ignition": "^2.0"
    },
    "repositories": {
        "flux-pro": {
            "type": "composer",
            "url": "https://composer.fluxui.dev"
        },
        "0": {
            "type": "composer",
            "url": "https://satis.ralphjsmit.com"
        },
        "1": {
            "type": "composer",
            "url": "https://satis.spatie.be"
        }
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        },
        "files": [
            "app\\helpers.php"
        ]
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi",
            "@php artisan filament:upgrade"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true,
            "php-http/discovery": true,
            "ralphjsmit/packages": true
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

---

### Additional Context

Any ideas on what else could be causing the issue? Any help is appreciated!

**Reply by sylainx** [reply]
Same for me.

I use Laravel 10.10

**Comment by sylainx** [comment]
I found a solution in the discord chat, but it not definitive


- I didnt publish filament views
- I didn't publish livewire views
- `composer update -W` doesn't work.

So, the solution I found is in this conversation https://discord.com/channels/883083792112300104/1322753981038067812/1322939944293109780

1. Update your composer.json to require `v3.5.12`

    `"livewire/livewire": "3.5.12"`

2. Run composer update for livewire 
`composer update livewire/livewire`

Once you’re back on v3.5.12, everything will be okay.

PS: This is not a definitive. Remember to modify your code once the problem has been solved

**Reply by studiowizjo** [reply]
Unfortunately this doesnt work for me.

**Reply by sylainx** [reply]
Can you show me your dependencies ?

**Reply by studiowizjo** [reply]
They are in comment above - https://github.com/filamentphp/filament/discussions/15091#discussioncomment-12043347

**Reply by sylainx** [reply]
Do exactly what I tell you below :

1.  Add this dependency in your `composer.json` file : `"livewire/livewire": "3.5.12"`
2. Remove your `vendor/` folder with this command `rm -r vendor./`
3. Run `composer install`
4. Check your dashboard now

**Reply by studiowizjo** [reply]
You are right, it works. Thank you :)!

**Reply by sylainx** [reply]
I'm glad for you, bro!
Make sure to change it once the issue is fixed.

**Reply by Kirad20** [reply]
ty, thats work for me

**Reply by sylainx** [reply]
> ty, thats work for me

🥳

**Reply by juanborras** [reply]
Works for me too!

**Comment by win00089** [comment]
good modal form view action work
